source/Calamari.Common/Features/ConfigurationVariables/ConfigurationVariablesReplacer.cs

```diff
@@ -48,8 +48,8 @@ public void ModifyConfigurationFile(string configurationFilePath, IVariables var
             {
                 if (ignoreVariableReplacementErrors)
                 {
-                    log.Warn(ex.Message);
-                    log.Warn(ex.StackTrace);
+                    log.InfoFormat("Ignoring variable replacement error: {0}", ex.Message);
+                    log.VerboseFormat(ex.StackTrace);
```

Comment:
I'm not sure if it matters, but should this just be `log.Verbose(ex.StackTrace);`

Reply:
Happy to approve if it doesn't and keep the green run.

Reply:
Probably doesn't matter but I'll do it right.